apps/condo/domains/miniapp/schema/B2BAppAccessRight.js

```diff
@@ -24,6 +24,16 @@ const B2BAppAccessRight = new GQLListSchema('B2BAppAccessRight', {
             kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
         },
     },
+    kmigratorOptions: {
```

Comment:
Should we add similar constraint to B2CApp as well?

Reply:
If we want to be consistent, then we should probably